script/integrate_and_upload.py

```
@@ -0,0 +1,198 @@
+# ----------------- ① 設定項目 -----------------
+# ハードコードされたキーの代わりに、環境変数から読み込む
+SUPABASE_URL = os.getenv("NEXT_PUBLIC_SUPABASE_URL")
+SUPABASE_KEY = os.getenv("NEXT_PUBLIC_SUPABASE_ANON_KEY")
+
+# スクリプトを実行する前に、キーが設定されているか確認する
+if not SUPABASE_URL or not SUPABASE_KEY:
+    print("エラー: SupabaseのURLまたはキーが.env.localファイルに設定されていません。")
+    print("プロジェクトのルートに.env.localファイルを作成し、以下の内容を記述してください。")
+    print("NEXT_PUBLIC_SUPABASE_URL=あなたのSupabaseのURL")
+    print("NEXT_PUBLIC_SUPABASE_ANON_KEY=あなたのSupabaseのANONキー")
+    exit() # キーがなければスクリプトを終了
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)